### ISSUE TITLE
Define Keybinding button does not have background in high contrast theme

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#50717]
Using the old keybindings editor:

![image](https://user-images.githubusercontent.com/2193314/40677084-7a69f0c0-6331-11e8-8995-65f152c408d9.png)

**Comment by sandy081** [comment]
@bpasero HC theme is not providing any `background-color` to buttons. Hence adding css style here because this button is special. 

Please let me know if we can solve it using themes.

**Comment by sandy081** [comment]
@bpasero With out background colour text gets overlapped with editor text. Please see in the picture in the description. 

Buttons in HC theme does not have background colour. See

![image](https://user-images.githubusercontent.com/10746682/40725958-02c0b8a2-6425-11e8-8831-e2d5f6dc4d12.png)

**Comment by bpasero** [comment]
@Tyriar we cannot have hardcoded colors in our CSS, we need to use theming colors. I see no reason why setting the background color to be `editor.background` theme color wouldn't work.
